### PR TITLE
Devl/functionality

### PR DESCRIPTION
**Pull request by kelvin-onlab**

### CODE REVIEW

File: TestON/tests/Sample/Sample.py [hunk 1]
```diff
@@ -0,0 +1,1451 @@
+
```
Comment (jhall11) [inline TestON/tests/Sample/Sample.py]: If you want to include this Sample test, could you provide some more description about what it is and maybe give it a more descriptive name?

File: TestON/tests/Sample/Sample.py [hunk 2]
```diff
@@ -0,0 +1,1451 @@
...
+        main.step( "Testing ONOS kill function" )
+        killResult = main.ONOSbench.onosKill( ONOS1Ip )
+
+        main.step( "Stopping ONOS service" )
+        stopResult = main.ONOSbench.onosStop( ONOS1Ip )
+
+        main.step( "Uninstalling ONOS service" )
+        uninstallResult = main.ONOSbench.onosUninstall()
+
+        case11Result = killResult and stopResult and uninstallResult
```
Comment (jhall11) [inline TestON/tests/Sample/Sample.py]: case15Result?

Comment (kelvin-onlab) [inline TestON/tests/Sample/Sample.py]: Ooops I don't want this.

File: TestON/tests/MultiProd/MultiProd.py [hunk 3]
```diff
@@ -53,18 +53,19 @@ def CASE1( self, main ):
...
         main.step( "Git checkout, pull and get version" )
-        #main.ONOSbench.gitCheckout( "master" )
+        main.ONOSbench.gitCheckout( "master" )
         gitPullResult = main.ONOSbench.gitPull()
         main.log.info( "git_pull_result = " + str( gitPullResult ))
         versionResult = main.ONOSbench.getVersion( report=True )
 
-        if gitPullResult == 1:
+        packageResult = main.TRUE
+        if gitPullResult == 100:
```
Comment (jhall11) [inline TestON/tests/MultiProd/MultiProd.py]: Is this ever true?

Comment (kelvin-onlab) [inline TestON/tests/MultiProd/MultiProd.py]: That was my purpose I do not want them to pull in different commit when
running in jenkins nightly, So the first test, Single 1.0, in this case
will pull the latest master and they will all run in that commit.

Comment (jhall11) [inline TestON/tests/MultiProd/MultiProd.py]: I would suggest making git pull configurable from the params file. We shouldn't assume these tests will always be run in a certain order.

Comment (kelvin-onlab) [inline TestON/tests/MultiProd/MultiProd.py]: I see that make sense. I'll push it in a bit
